Assets/Scripts/MainMenu.cs

```diff
@@ -0,0 +1,89 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class MainMenu : MonoBehaviour
+{
+    private string XInput, AInput;
+    private int button;
+    [SerializeField] GameObject button0, button1;
+    [SerializeField] GameObject mainMenuObj;
+    [SerializeField] GameObject customizeMenuObj;
+    private Image image0, image1;
+    public bool main;
+    public CustomizeMenu customizeMenuScript;
+    private float timer;
+    private float waitTime = 0.2f;
+    void Start()
+    {
+        image0 = button0.GetComponent<Image>();
+        image1 = button1.GetComponent<Image>();
+        XInput = "js2";
+        AInput = "js10";
+        button = 0;
+        // image0.color = Color.yellow;
```

Comment:
Should also be kept to avoid confusing player (let them know what button is first)